src/content/ui-components/install/laravel.mdx

```diff
@@ -0,0 +1,60 @@
+---
+title: Laravel
+meta:
+  title: Laravel | Tiptap UI Components
+  description: Install and configure Tiptap UI Components in your Laravel project.
+  category: UI Components
+---
+
+Use the command below to scaffold a new Laravel project with React support:
+
+```bash
+laravel new my-app --react
+```
+
+## Add Tiptap Components
```

Comment:
In the Tiptap docs, titles go in sentence case (only first letter is capitalized, except if it's proper nouns.